src/utils.jl

```diff
@@ -10,6 +10,23 @@ macro check_args(K, param, cond, desc=string(cond))
 end
 
 
+"""
+    ColVecs{T, TX<:AbstractMatrix}
```

Comment:
Maybe just
```suggestion
    ColVecs
```
or add the third type parameter?